nix/default.nix

```diff
@@ -192,6 +192,7 @@ let
         mkdir $out
         mv output $out/
         cp -r $src/* $out/
+        cp -r $src/src $out/
```

Comment:
Please remove this change. This isn't related to the docs search (the `installPhase` is overridden) and is the reason CI is failing